websites/K/KaniWork/dist/metadata.json

```diff
@@ -4,27 +4,28 @@
     "name": "fantomitechno 🦊",
     "id": "563749920683720709"
   },
-  "url": "kanishiel.com",
+  "url": "kaniwork.com",
   "description": {
-    "en": "KaniShiel is a discord bot that can host multiple games on an indefinite number of simultaneous servers.",
-    "fr": "KaniShiel est un bot de jeu qui peut animer de multiple jeu sur un nombre indéfini de serveur simultané."
+    "en": "The Kani Network is a community gathered around different discord bots",
+    "fr": "Le Kani Network est une communauté réunis autour de différents bots discord"
   },
-  "service": "KaniShiel",
-  "version": "1.2.1",
+  "service": "KaniNetwork",
+  "version": "1.3.0",
   "logo": "https://i.imgur.com/hCHdlGM.png",
   "thumbnail": "https://i.imgur.com/zqESW36.png",
   "color": "#FF0000",
   "tags": [
     "bot",
     "kanishiel",
     "discord",
-    "game-bot"
+    "game-bot",
+    "kani-network"
   ],
   "category": "other",
   "settings": [
     {
       "id": "privacy",
-      "title": "Hide server",
+      "title": "Hide server in the kanishiel's dashboard",
```

Comment:
Isn't this too long

Reply:
I think yes and the old fit with the changes so why I changed this 🤔 I take it back